tsconfig.json

```diff
@@ -9,8 +9,6 @@
     "forceConsistentCasingInFileNames": true,
     "lib": ["es2020"],
     "module": "node16",
-    "newLine": "lf",
```

Comment:
this is now the default https://devblogs.microsoft.com/typescript/announcing-typescript-5-0/#deprecations-and-default-changes